src/index.html

```diff
@@ -9,10 +9,46 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <section
+        class="calendar calendar--start-day-sun calendar--month-length-31"
+      >
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
```

Comment:
Try to use ul and li in this case.
it's preferred because we have one repeated element inside div.
In this case best variant use list)